ister.py

```diff
@@ -1418,6 +1423,9 @@ def handle_options():
     parser.add_argument("-S", "--statedir", action="store",
                         default="/var/lib/swupd",
                         help="Path to swupd state dir")
+    parser.add_argument("-F", "--fast-install", action="store_true",
```

Comment:
To avoid confusing users, please put this in a [mutual-exclusion](https://docs.python.org/3/library/argparse.html#mutual-exclusion) with the statedir argument.

Reply:
I forgot to do this.  I'll update the PR.